src/tinypeel/Peeling/PeelingIO.py

```diff
@@ -68,7 +68,9 @@ def writeOutParamaters(peelingInfo):
 
     np.savetxt(args.out + ".geno_error_prob.txt", peelingInfo.genoError, fmt="%f")
     np.savetxt(args.out + ".seq_error_prob.txt", peelingInfo.seqError, fmt="%f")
-    np.savetxt(args.out + ".rec_prob.txt", np.empty((1,1)), fmt="%f") # not be realized, just as a placeholder
+    np.savetxt(
```

Comment:
@AprilYUZhang as mentioned in another comment - we should flag this line in an issue so we get back to it at some point